ascon-aead/Cargo.toml

```diff
@@ -20,22 +20,23 @@ subtle = { version = "2", default-features = false }
 zeroize = { version = "1.6", optional = true, default-features = false, features = [
     "derive",
 ] }
-ascon = "0.3"
+ascon = "0.4"
 
 [dev-dependencies]
 hex = "0.4"
 aead = { version = "0.5", features = ["alloc"] }
 spectral = { version = "0.6", default-features = false }
 
 [features]
-default = ["alloc", "getrandom", "zeroize"]
+default = ["alloc", "getrandom", "zeroize", "sponge-zeroize"]
 std = ["aead/std", "alloc"]
 alloc = ["aead/alloc"]
 arrayvec = ["aead/arrayvec"]
 getrandom = ["aead/getrandom", "rand_core"]
 heapless = ["aead/heapless"]
 rand_core = ["aead/rand_core"]
 stream = ["aead/stream"]
+sponge-zeroize = ["ascon/zeroize"]
```

Comment:
I think you can simply use `zeroize ="ascon/zeroize"]`.

Reply:
As `ascon-aead` has an optional `zeroize` dependency, there is a naming conflict.

`dep:` would solve this, but is only available with 1.60 or later. The current MSRV of `ascon-aead` is 1.56.

Reply:
One option is to rename the zeroize crate with a `package` attribute, e.g.:

```rust
zeroize_crate = { package = "zeroize", ... }
```

Alternatively we could bump MSRV/version, although that'd be easier to do to the entire repo at the same time.

Reply:
I'd prefer the first option and delay the change to the MSRV until it's performed for the entire repo.

Reply:
Thanks for the suggestion :+1: I updated it accordingly